app/src/main/java/org/p2p/wallet/bridge/api/request/SendEthereumBundleRpcRequest.kt

```diff
@@ -7,7 +7,7 @@ import org.p2p.wallet.bridge.model.BridgeBundle
 data class SendEthereumBundleRpcRequest(
     @Transient val bundleRequest: BridgeBundle,
 ) : JsonRpc<Map<String, Any>, Unit>(
-    method = "send_ethereum_bundle",
+    method = "send_ethereum_bundle", // "send_ethereum_bundle",
```

Comment:
why comment?

Reply:
will be switched on the release method on release